backend/tests/fixtures/filter/FilterServicMock.ts

```diff
@@ -0,0 +1,39 @@
+import { FilterServiceBase } from '../../../src/filter/filter.service.base';
+import {
+  EdgeTypeFilterModel,
+  NodeTypeFilterModel,
+} from '../../../src/shared/filter';
+import {
+  getEdgeTypeFilterModelResult,
+  getNodeTypeFilterModelResult,
+} from './FilterQueryResults';
+
+export default class FilterServiceMock implements FilterServiceBase {
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
disable this globally as described in some other pull request as well

Reply:
I openend  #142 to track this.